src/client/rest_client/projectsClient.js

```diff
@@ -154,7 +154,7 @@ class ProjectsClient extends BaseClient {
     getLatestCommits(ownerId, projectName, numCommits) {
         var query;
         numCommits = numCommits || 100;
-        query = '?n=' + numCommits;
+        query = 'commits?n=' + numCommits;
         return super.get('projects/' + ownerId + '/' + projectName, query);
```

Comment:
https://editor.dev.webgme.org/api/projects/demo/FSM/commits/?n=1
also works so even if superagent adds an extra '/' we should use the commits as part of the path